lib/kennel.rb

```diff
@@ -70,7 +70,8 @@ def store(parts)
           path = "generated/#{part.tracking_id.tr("/", ":").sub(":", "/")}.json"
           used << File.dirname(path) # only 1 level of sub folders, so this is safe
           used << path
-          write_file_if_necessary(path, JSON.pretty_generate(part.as_json) << "\n")
+          payload = part.as_json.merge(api_resource: part.class.api_resource)
```

Comment:
FYI this is faster / does not allocate a new hash
```suggestion
          payload = part.as_json
          payload[:api_resource] = part.class.api_resource
```

Reply:
... I think I broke it by adding that 🤦

Reply:
Yes, because you're mutating the to_json hash, which something else then detects and complains about